src/frontend/komponenter/Behandling/BehandlingContainer.tsx

```diff
@@ -56,6 +57,13 @@ const BehandlingContainer: FC = () => {
                             from="/behandling/:behandlingId/"
                             to="/behandling/:behandlingId/inngangsvilkar"
                         />
+                        <Route
+                            exact={true}
+                            path="/behandling/:behandlingId/personopplysninger"
+                            render={() => {
+                                return <Personopplysninger id={'1234'} />;
```

Comment:
Husk å sende med behandlingId til `Personopplysninger`?

Reply:
Trenger vi behandlingId i ```Personopplysninger```?

Reply:
Kommenterte på slack https://nav-it.slack.com/archives/G012YEK9YQ1/p1605539032341100?thread_ts=1605535386.340900&cid=G012YEK9YQ1

Reply:
Nei, godt poeng. Er vel kanskje fnr vi er mest interessert i når det gjelder personopplysninger-siden?

Reply:
Vi har vel inte fnr i detta tidspunktet ? 🤔

Reply:
Som Alexandra påpeker har vi ikke fnr når vi rendrer personopplysninger, så endrer til behandlingId.